lib/src/components/NxTextInput/NxTextInput.tsx

```diff
@@ -61,19 +63,26 @@ const NxTextInput = forwardRef<HTMLInputElement | HTMLTextAreaElement, Props>(
       }
 
       return (
-        <NxTooltip open={isInvalid}
-                   title={firstValidationError}
-                   className="nx-tooltip--validation-error"
-                   placement="top-end">
-          {React.createElement(element, {
-            ...newProps,
-            ref,
-            type: typeAttr,
-            onChange: inputOnChange,
-            className: internalClassName,
-            onKeyPress: inputOnKeyPress
-          })}
-        </NxTooltip>
+        <div className={internalClassName}>
+          <div className="nx-text-input__box">
+            {React.createElement(element, {
+              ...newProps,
+              disabled,
+              ref,
+              type: typeAttr,
+              onChange: inputOnChange,
+              className: "nx-text-input__input",
+              onKeyPress: inputOnKeyPress
+            })}
+            <span className="nx-text-input__validation-icon">
+              <NxFontAwesomeIcon icon={faCheck} className="nx-icon--valid"/>
+              <NxFontAwesomeIcon icon={faExclamationCircle} className="nx-icon--invalid"/>
+            </span>
+          </div>
+          <div className="nx-text-input__invalid-message">
+            <span className="nx-text-input__invalid-message-text">{firstValidationError} &nbsp;</span>
```

Comment:
I think this should work:
```suggestion
            <span className="nx-text-input__invalid-message-text">{firstValidationError || '&nbsp;'}</span>
```